core/src/main/java/hudson/model/Executor.java

```diff
@@ -594,6 +653,32 @@ protected Object call(Object o, Method m, Object[] args) throws Throwable {
     }
 
     /**
+     * Finds the executor running a given process.
```

Comment:
Finds the executor that was running a given process while this method was being called. There is the potential that the process may complete by the time this method returns and thus a non-null result could correspond to an Executor running a different process